### PR TITLE
155136904 download error feedback

### PR DESCRIPTION
**Pull request by CodingCarlos**

### CODE REVIEW

File: pages/video-download/video-download.view.scss [hunk 1]
```diff
@@ -162,3 +162,10 @@ a:hover {
 		text-align: center;
 	}
 }
+
+.code-error {
+	padding: 8px;
+	display: inline-block;
+	width: 50%;
+	color: $error;
```
Comment (javierLiarte) [inline pages/video-download/video-download.view.scss:176]: sass doesnt find $error

File: pages/video-download/video-download.view.html [hunk 2]
```diff
@@ -1,5 +1,18 @@
 <div ng-controller="VideoDownloadController as VideoDownload" class="download-page">
-	<article>
+
+	<!-- Screen loading directive -->
+	<div ng-if="!VideoDownload.video.data && VideoDownload.loading" class="loading">
```
Comment (javierLiarte) [inline pages/video-download/video-download.view.html:4]: maybe a 404 error directive here for handling such 404 backend errors and having consistent errors accross all the platform?

**Review by javierLiarte — APPROVED** [review]
sass reports an error on build, the rest seems ok, and the page shows feedback to user

also, we have to increment a patch number in package.json version

 ---> Running in d520cd627ad6
[07:18:10] Working directory changed to /app/src
[07:18:10] Using gulpfile /app/src/gulpfile.js
[07:18:10] Starting 'make-config'...
[07:18:10] Starting 'js'...
[07:18:10] Finished 'js' after 11 ms
[07:18:10] Starting 'html'...
[07:18:10] Finished 'html' after 1.27 ms
[07:18:10] Starting 'tests'...
[07:18:10] Finished 'tests' after 1.68 ms
[07:18:10] Starting 'sass'...
[07:18:10] Finished 'sass' after 4.23 ms
[07:18:10] Finished 'make-config' after 109 ms
[07:18:10] Starting 'build'...
[07:18:10] Finished 'build' after 31 μs
[07:18:10] Join JS: js/app.js
[07:18:10] Join tests: test-globals.js
Error in plugin 'sass'
Message:
    pages/video-download/video-download.view.scss
Error: Undefined variable: "$error".
        on line 170 of pages/video-download/video-download.view.scss
>> 	color: $error;
   --------^

[07:18:11] Move SCSS: 0 items
[07:18:11] Join JS: js/config.js
[07:18:11] Move HTML: components/loading/loading.view.html
[07:18:11] Join tests: pages/login/login.controller.spec.js
[07:18:11] Join JS: languages/en_us.js
[07:18:11] Move HTML: pages/gallery/gallery.view.html
[07:18:11] Join tests: pages/login/login.service.spec.js
[07:18:11] Join JS: languages/es_es.js
[07:18:11] Move HTML: pages/home/home.view.html
[07:18:11] Join tests: pages/register/register.controller.spec.js
[07:18:11] Join JS: services/api.js
[07:18:11] Move HTML: pages/login/login.view.html
[07:18:11] Join tests: pages/register/register.service.spec.js
[07:18:11] Join tests: 5 items
[07:18:11] Join JS: services/session.js
[07:18:11] Move HTML: pages/register/register.view.html
[07:18:11] Join JS: services/video.js
[07:18:11] Move HTML: pages/terms/terms.view.html
[07:18:11] Join JS: components/loading/loading.js
[07:18:11] Move HTML: pages/video-download/video-download.view.html
[07:18:11] Move HTML: 7 items
[07:18:11] Join JS: pages/gallery/gallery.controller.js
[07:18:11] Join JS: pages/gallery/gallery.service.js
[07:18:11] Join JS: pages/login/login.controller.js
[07:18:11] Join JS: pages/login/login.service.js
[07:18:11] Join JS: pages/register/register.controller.js
[07:18:11] Join JS: pages/register/register.service.js
[07:18:11] Join JS: pages/video-download/video-download.controller.js
[07:18:11] Join JS: pages/video-download/video-download.service.js
[07:18:11] Join JS: 16 items
Removing intermediate container d520cd627ad6

File: pages/video-download/video-download.view.scss (hunk 1 above)
Comment (CodingCarlos) [reply pages/video-download/video-download.view.scss:176]: Ups! My fail. That variable was defined in #27 T_T We could mege this into #27, and that one into dev, or something like that

File: pages/video-download/video-download.view.html (hunk 2 above)
Comment (CodingCarlos) [reply pages/video-download/video-download.view.html:4]: I don't know if we shall have one only 404 error page, or have different ones for each type of content (such as, for example, [youtube does](https://www.youtube.com/watch?v=oTugqOT0))

Comment (javierLiarte) [reply pages/video-download/video-download.view.html:4]: lets talks about later with pablo and juan, two approaches are commonly seen in other platforms

File: pages/video-download/video-download.view.scss (hunk 1 above)
Comment (javierLiarte) [reply pages/video-download/video-download.view.scss:176]: #27 is now conflicting with dev after merging #28 should be updated first, and some review would be desirable, maybe @DevStarlight could help

**Review by DevStarlight — APPROVED** [review]

**Review by DevStarlight — APPROVED** [review]

**Comment by javierLiarte** [comment]
@DevStarlight we  missed to upgrade version number in this PR!!

from my first comment: "also, we have to increment a patch number in package.json version"

**Comment by javierLiarte** [comment]
Setup for version v0.0.4 fixed in 4f29b24

**Comment by DevStarlight** [comment]
@javierLiarte I was coming to fix it right now but I saw you already upgraded that. Thanks.

**Pull request merged**